code/tick/feed.q

```diff
@@ -84,4 +84,4 @@ init:{
 h:.servers.gethandlebytype[`tickerplant;`any]
 
 init 0
-.z.ts:feed
+.timer.repeat[.proc.cp[];0Wp;0D00:00:00.200;(`feed;`);"Publish Feed"];
```

Comment:
I'm not sure myself. Going on comments in  timer.q claiming it is the safest option. Though I am getting errors popping up now, so I am going to close the request until I can figure it out.